src/appState/AppState.ts

```diff
@@ -38,8 +38,10 @@ export type ViewSplit = typeof viewSplits[number]
 
 export interface Settings {
   autoUpdateApp: boolean;
+  extensions?: Record<string, boolean>;
```

Comment:
Could we use the type we have from `markdown-it-pandoc` here instead of `Record<string, boolean>`?
Also, seems `extensions` can never be undefined, so we don't need the `?`

Reply:
Ah. Of course. That's an excellent idea... However, it's not exported yet. So here's another PR over there: https://github.com/mb21/markdown-it-pandoc/pull/15